### DISCUSSION TITLE
Update to new version

### DISCUSSION THREAD

**Discussion by minershaven** [coinfoundry/miningcore#1332] [Q&A] [answered]
what is the easiest way to update miningcore when a new version is released?  Wipe out the miningcore folder and just build the new version ?

**Comment by seperatis** [comment, marked as answer]
Build the latest, make sure to **save your pool configs**, and pay attention if any database updates need to be made. If the new release requires database modifications, Oliver will _say so_ in the release explicitly. _Most_ releases **do not** require database changes.

**Reply by minershaven** [reply]
Thanks.  That is what i figured.  saved my config.json file and just rebuilt it.  Just started wondering if their was an easier or faster way as i sat here watching things scroll through putty.
